src/components/fields/edit/TextEditor.jsx

```diff
@@ -37,7 +42,11 @@ const TextInput = function(props) {
         id={schema.name}
         multiline={isLongString}
         maxRows={isLongString ? 5 : undefined}
-        label={label}
+        label={
+          placeholder
```

Comment:
it doesn't really make sense to have a `placeholder` prop that takes precedence over label. if anything, i would expect placeholder to be a fallback if label was absent. probably the best way to do this is to change the labelId that is passed into the TextEditor component rather than changing TextEditor